src/condor_credd/condor_credmon_oauth/examples/config/condor/40-oauth-credmon.conf

```diff
@@ -4,14 +4,17 @@
 #
 use feature : OAUTH
 
-# MANDATORY for local credmon: Uncomment this line to have submitters
-# always create a credential (may conflict with Kerberos credmon)
+# MANDATORY for local credmon: Uncomment these lines to have submitters always
+# locally create a scitokens credential.  Job submit files then need to have
+#    use_oauth_services = scitokens
+# to send the credentials to the jobs.
 #
-# SEC_CREDENTIAL_PRODUCER = /usr/sbin/scitokens_credential_producer
+# LOCAL_CREDMON_PROVIDER_NAME = scitokens
+# SEC_PROCESS_SUBMIT_TOKENS = false
 
 # MANDATORY for enabling the transfer of credentials from submit host
 # to execute hosts, if encryption is not already enabled.
-# SEC_DEFAULT_ENCRYPTION = REQUIRED
+SEC_DEFAULT_ENCRYPTION = REQUIRED
```

Comment:
Going to disagree here, only because setting `SEC_DEFAULT_ENCRYPTION = REQUIRED` is a quick way to take down your schedd if your pool isn't ready for it. If an admin forgets to set this but otherwise sets things up, tokens won't work and hopefully they see the messages in the CredLog and CredMonOAuthLog as to why (no encryption).